docs/api-reference/lab-testing-insurance/search-diagnosis.mdx

```diff
@@ -0,0 +1,37 @@
+---
+openapi: "GET /v3/insurance/search/diagnosis"
+---
+
+<RequestExample>
+
+```bash cURL
+curl --request GET \
+     --url '{{BASE_URL}}/v3/insurance/search/diagnosis?diagnosis_query=glucose' \
+     --header 'accept: application/json' \
+     --header 'x-vital-api-key: {YOUR_KEY}'
+```
```

Comment:
Usually we put `cURL`, `Python` and `Node` examples. Unless the last two are not available yet?

Reply:
We still don't have it. I'll add a ticket to Linear for this.